ProcessMaker/Cache/Settings/SettingCacheManager.php

```diff
@@ -108,6 +112,34 @@ public function clear(): bool
         return $this->cacheManager->flush();
     }
 
+    /**
+     * Remove items from the settings cache by a given pattern.
+     *
+     * @param string $pattern
+     *
+     * @throws \Exception
+     * @return void
+     */
+    public function clearBy(string $pattern): void
+    {
+        try {
+            // get the connection name from the cache manager
+            $connection = $this->cacheManager->connection()->getName();
+            // Get all keys
+            $keys = Redis::connection($connection)->keys('*');
```

Comment:
What if the cache drive is not Redis?
Consider to close the Settings driver `CACHE_SETTING_DRIVER` to be always `cache_settings` or implement a general logic for other types of drivers.

Reply:
A validation was added